Examples/Monocular/mono_kitti.cc

```diff
@@ -32,25 +32,34 @@
 using namespace std;
 
 void LoadImages(const string &strSequence, vector<string> &vstrImageFilenames,
-                vector<double> &vTimestamps);
+                vector<double> &vTimestamps, int start_frame=0, int end_frame=-1);
 
 int main(int argc, char **argv)
 {
-    if(argc != 5)
+    if(argc <= 5)
```

Comment:
Should this be something like
```c++
if(argc < 5 or 7 < argc)
```

Looks like this will currently throw an error with only the required arguments

Reply:
You are correct... great catch! Fixed and made it just `< 5` since the 6th and 7th arguments are optional. Anything beyond 7 will be ignored.